### PR TITLE
feat: Additional files for transform

### PR DESCRIPTION
**Pull request by doosuu**

### CODE REVIEW

File: src/modules/setup.ts [hunk 1]
```diff
@@ -41,30 +43,30 @@ class ReplaceVariablesStream extends Transform {
...
+                noticeComment = `<!-- ${NOTICE_COMMENT} -->`;
                 const xmlDeclarationArray = xmlDeclarationRegExp.exec(result);
                 if (xmlDeclarationArray !== null && result.startsWith(xmlDeclarationArray[0])) {
                     result = this._injectNoticeAfterStartLine(result, xmlDeclarationArray[0], noticeComment);
                 } else {
                     result = `${noticeComment}\n${result}`;
                 }
-            } else if (['.yaml', '.yml', '.sh'].includes(this._fileExt)) {
-                noticeComment = `# ${notice}`;
+            } else if (['Dockerfile'].includes(this._filename) || ['.yaml', '.yml', '.sh', '.dockerfile'].includes(fileExt)) {
```
Comment (dennismeister93) [inline src/modules/setup.ts:61]: with this we would not allow people to have for example something like
"Dockerfile-test"
"Dockerfile-prod"
etc.

**Review by dennismeister93 — CHANGES_REQUESTED** [review]

File: src/modules/setup.ts [hunk 2]
```diff
@@ -13,70 +13,170 @@
...
+                (Array.isArray(filename) && filename.includes(this._filename));
+        }
+        return filenameMatches;
+    }
+
+    private _isKnownFile(transformableFiletype: FileTypeInformation): boolean {
+        return this._hasMatchingFileExtension(transformableFiletype) || this._hasMatchingFilename(transformableFiletype);
+    }
+
+    canHandleFile(): boolean {
```
Comment (dennismeister93) [inline src/modules/setup.ts:112]: add visibility public and move below private methods

File: src/modules/setup.ts [hunk 3]
```diff
@@ -13,70 +13,170 @@
...
-            } else if (['.json'].includes(this._fileExt)) {
-                noticeComment = `// ${notice}`;
-                result = `${noticeComment}\n${result}`;
             }
+        }
+        return insertionLine;
+    }
 
-            this._firstChunk = false;
+    private _insertCommentAfterLine(textChunk: string, startingLine: string, noticeComment: string) {
```
Comment (dennismeister93) [inline src/modules/setup.ts:143]: return type missing

**Review by dennismeister93 — CHANGES_REQUESTED** [review]

File: src/modules/setup.ts (hunk 2 above)
Comment (doosuu) [reply src/modules/setup.ts:112]: Done

File: src/modules/setup.ts (hunk 3 above)
Comment (doosuu) [reply src/modules/setup.ts:143]: done

**Review by dennismeister93 — APPROVED** [review]
lgtm

**Pull request merged**